rest-server/test/jobDetail.js

```diff
@@ -0,0 +1,237 @@
+            'applicationExitType': 'UNKNOWN'
+          },
+          'aggregatedTaskRoleStatuses': {
+            'ptb_instance': {
+              'taskRoleStatus': {
+                'taskRoleName': 'ptb_instance',
+                'taskRoleRolloutStatus': {
+                  'overallRolloutServiceVersion': null,
+                  'overallRolloutStatus': 'UNKNOWN',
+                  'overallRolloutStartTimestamp': null,
+                  'overallRolloutEndTimestamp': null,
+                  'currentRolloutScaleUnit': null,
+                  'currentRolloutTaskIndexes': null,
+                  'currentRolloutStatus': 'UNKNOWN',
+                  'currentRolloutStartTimestamp': null,
+                  'currentRolloutEndTimestamp': null
+                },
+                'frameworkVersion': 10
+              },
+              'taskStatuses': {
+                'taskRoleName': 'ptb_instance',
+                'taskStatusArray': [
+                  {
+                    'taskIndex': 0,
+                    'taskRoleName': 'ptb_instance',
+                    'taskState': 'TASK_COMPLETED',
+                    'taskRetryPolicyState': {
+                      'retriedCount': 0,
+                      'transientNormalRetriedCount': 0,
+                      'transientConflictRetriedCount': 0,
+                      'nonTransientRetriedCount': 0,
+                      'unKnownRetriedCount': 0
+                    },
+                    'taskCreatedTimestamp': 1522559440659,
+                    'taskCompletedTimestamp': 1522560381565,
+                    'taskServiceStatus': {
+                      'serviceVersion': 0
+                    },
+                    'containerId': 'container_e11_1522466352615_0006_01_000002',
+                    'containerHost': '10.151.40.131',
+                    'containerIp': '10.151.40.131',
+                    'containerGpus': 0,
+                    'containerLogHttpAddress': 'http://10.151.40.131:8042/node/containerlogs/container_e11_1522466352615_0006_01_000002/admin/',
```

Comment:
It is not necessary to include such details. And you may want to make use of the `frameworkDetailTemplate` defined in `setup.js`.